cmd/demo/opasupport/opa_support.go

```diff
@@ -62,8 +62,12 @@ func (o *OpaSupport) Middleware(next http.Handler) http.Handler {
 				return
 			}
 		}
+		scheme := "http"
```

Comment:
Not sure if this is possible but didn't know if we should default to `http` if there is no scheme detected in the url.